darts/models/torch_forecasting_model.py

```diff
@@ -541,71 +563,108 @@ def predict_from_dataset(self,
         with torch.no_grad():
             for batch_tuple in iterator:
 
-                input_series, indices = batch_tuple[0], batch_tuple[-1]
+                input_series = batch_tuple[0]
```

Comment:
Could you maybe add a comment to remind here that `input_series` already contains the past covariates concatenated to the target series

Reply:
Done